hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/deltacommit/SparkUpsertDeltaCommitPartitioner.java

```diff
@@ -73,13 +74,13 @@ protected List<SmallFile> getSmallFiles(String partitionPath) {
       if (smallFileSlice.getBaseFile().isPresent()) {
         HoodieBaseFile baseFile = smallFileSlice.getBaseFile().get();
         sf.location = new HoodieRecordLocation(baseFile.getCommitTime(), baseFile.getFileId());
-        sf.sizeBytes = getTotalFileSize(smallFileSlice);
+        sf.sizeBytes = FileSliceUtils.getTotalFileSizeAsParquetFormat(smallFileSlice, config.getLogFileToParquetCompressionRatio());
```

Comment:
Can be moved to class `FileSlice`, and keep the comment of method: `convertLogFilesSizeToExpectedParquetSize`.

Reply:
> Can be moved to class `FileSlice`, and keep the comment of method: `convertLogFilesSizeToExpectedParquetSize`.

done~